docs/_data/navigation.yml

```diff
@@ -0,0 +1,51 @@
+header:
+  - title: Book
+    url: https://redhat-appstudio.github.io/book/
+  - title: Github
+    url: https://github.com/redhat-appstudio/infra-deployments
+
+docs-en:
+  - title: Introduction
```

Comment:
it will require overriding some of the templates of the theme I'm using (I didn't find a better template than the one I used). I think that we can proceed with the current YAML that declares the order of the sidebar and if we see it causes maintenance issues we can consider implementing another approach.